administrator/language/de-DE/lib_joomla.ini

```
@@ -496,6 +496,8 @@ JLIB_HTML_FEATURED_PENDING_ITEM="Hauptbeitrag, aber zurückgestellt."
 JLIB_HTML_FEATURED_STARTED="Start: %s"
 JLIB_HTML_GOTO_PAGE="Gehe zur Seite %s"
 JLIB_HTML_GOTO_POSITION="Zur Seite %s wechseln"
+JLIB_HTML_ITEM_PUBLISHED_BUT_CATEGORY_TRASHED="Veröffentlicht, aber die Kategorie wurde in den Papierkorb verschoben."
```

Sollte auch hier die Gegenwart verwendet werden (ist ja auch noch immer im Papierkorb)?
```suggestion
JLIB_HTML_ITEM_PUBLISHED_BUT_CATEGORY_TRASHED="Veröffentlicht, aber die Kategorie ist im Papierkorb."
```

---

Hört sich komisch an und es war ja definitiv davor.